dev_guide/deployments.adoc

```diff
@@ -381,3 +382,17 @@ Rollbacks revert an application back to a previous deployment and can be
 performed using the REST API or the CLI. See the
 link:../cli_reference/basic_cli_operations.html#deployment-operations[CLI
 Reference] for more details.
+
+== Resizing
+In addition to rollbacks, you can can exercise fine-grained control over
+the number of replicas by using the `osc resize` command.
+For example, the following command sets the replicas in the deployment
+configuration `frontend` to 3.
+
+----
+$ osc resize dc frontend --replicas=3
+----
+
+The number of replicas will also propagate to the desired state and,
+eventually, current state of the deployment configured by the deployment
```

Comment:
I'm not sure about all those commas.

I think you could write this, which would flow smoother: "The number of replicas also propagates to the desired state, and eventually to the current state of the deployment, as configured by the deployment configuration frontend."

As it is now, it stops/starts a bit abruptly. But this is just a nitpick.